modules/limelightDigitalBidAdapter.js

```diff
@@ -55,8 +55,9 @@ export const spec = {
       winTop = window;
     }
     const placements = groupBy(validBidRequests.map(bidRequest => buildPlacement(bidRequest)), 'host')
-    return Object.keys(placements)
+    let map = Object.keys(placements)
       .map(host => buildRequest(winTop, host, placements[host].map(placement => placement.adUnit)));
+    return map;
```

Comment:
This change seems unnecessary. Is there a particular reason you included it?

Reply:
fixed